commands/earnings.js

```diff
@@ -19,7 +19,8 @@ const earnings = (reply, message) => {
             `${hours} Hour Profitability for *${format.hashrate(hashes)}*:\n` +
                 `- BTCZ: *${format.coins(BTCZEarnings)}*\n` +
                 `- BTC: *${format.coins(BTCEarnings)}*\n` +
-                `- USD: *${format.usd(USDEarnings)}*`
+                `- USD: *${format.usd(USDEarnings)}*\n` +
```

Comment:
So, I think the back-ticks let you do multiple lines without having to concatenate. It's just weird to read it this way.

Reply:
no, backticks allow me to not have to `' + '`

Backticks actually allow you to do multiple lines but I'd have to shift everything to the left which I thought wouldn't be as readable.

Reply:
`- USD: *${format.usd(USDEarnings)}*\n` using quotes translates to:
`- USD: *' + format.usd(USDEarnings) + '*\n'`

Reply:
Oh, I misread your "I think" comment -- but yeah, shifting everything to the left looked weird IMHO. I can shift it if you wish.

Reply:
Nah. Just ship it.